deepchem/models/torch_models/tests/test_grover.py

```diff
@@ -1,27 +0,0 @@
-try:
-    import torch
-except ModuleNotFoundError:
-    pass
-
-
-def testGroverEmbedding():
-    from deepchem.models.torch_models.grover_layers import GroverEmbedding
-    hidden_size = 8
-    f_atoms = torch.randn(4, 151)
-    f_bonds = torch.randn(5, 165)
-    a2b = torch.Tensor([[0, 0], [2, 0], [4, 0], [1, 3]]).type(torch.int32)
-    b2a = torch.Tensor([0, 1, 3, 2, 3]).type(torch.long)
```

Comment:
This test looks slightly longer than the duplicate test. Are there any meaningful difference between this test and the one in test_grover_layers?

Reply:
The values in `test_grover` are randomly initialized while the values in `test_grover_layers` are from featurized data, so we don't have random initialization there.